SmartDeviceLink/SDLLockScreenViewController.m

```diff
@@ -39,6 +39,10 @@ - (BOOL)shouldAutorotate {
     return NO;
 }
 
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
+    return UIInterfaceOrientationMaskPortrait;
+}
+
```

Comment:
Sometimes the app crashes right after connection to the SDL accessory due to the background color of the lock screen not being set on the main thread. The crash occurs in the `setBackgroundColor:backgroundColor` function.

Reply:
Should be fixed

Reply:
This seems to be fixed. I haven't had an issue since the fix was pushed.